package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "reports-in-typescript",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/index.js",
+  "scripts": {
+    "start:dev": "nodemon --watch",
```

Comment:
Is there a reason you chose not to include a start script for a 'production release' of the application?